velox/tool/trace/TableScanReplayer.cpp

```diff
@@ -54,11 +54,11 @@ std::vector<exec::Split> TableScanReplayer::getSplits() const {
   std::vector<std::string> splitInfoDirs;
   if (driverId_ != -1) {
     splitInfoDirs.push_back(exec::trace::getOpTraceDirectory(
```

Comment:
Can we VELOX_CHECK that non-hash join replay only expects one pipeline and hash join has two pipeline?

Reply:
add check in OperatorReplayerBase
```C++
if (operatorType_ == "HashJoin") {
    VELOX_USER_CHECK_EQ(pipelineIds_.size(), 2);
  } else {
    VELOX_USER_CHECK_EQ(pipelineIds_.size(), 1);
  }
```